src/AvaloniaCoreRTDemo.csproj

```diff
@@ -2,25 +2,31 @@
 
   <PropertyGroup>
     <!-- if you want to check exceptions, change this to "Exe" and start your app from the console -->
-    <OutputType>WinExe</OutputType>
+    <OutputType Condition="'$(Configuration.toUpper())' != 'DEBUG'">WinExe</OutputType>
+    <OutputType Condition="'$(Configuration.toUpper())' == 'DEBUG'">Exe</OutputType>
```

Comment:
Why is this change needed? I prefer debug to be as close to release as possible, but maybe I am missing something?

Reply:
The project references to Avalonia.Diagnostics for debugging purposes only, I tried to extend that concept to IlcGenerateStackTraceData, StripSymbols and OutputType. That's useful for getting errors (especially those related to missing metadata), for once you resolve them, you may not need the console. When you use WinExe, there is no console in Windows, so these errors messages are lost by default.

Reply:
I agree that having a console window open when debugging is very useful. I don't need to separately launch the console and then launch an app in order to see what types are missing etc.